src/main/java/org/jenkinsci/plugins/gitclient/CliGitAPIImpl.java

```diff
@@ -575,7 +575,7 @@ public MergeCommand setStrategy(MergeCommand.Strategy strategy) {
                 this.strategy = strategy.toString();
                 return this;
             }
-
+            
```

Comment:
Please remove this pure white space change.  Formatting is already bad enough in the plugin without introducing more lines with trailing white space.